### ISSUE TITLE
Suggestion - BlackMarket

### ISSUE THREAD

**Issue by Skai97** [InZidiuZ/op-framework-issue-tracker#3103]
### Assurance

I have read the `README.md`

### Summary

Black market that utilizes cryptocurrency obtained through boosting, or another possible method for generating crypto (e.g., selling to the black market, Mining rig? (built inside of home breaks after x amount of time) 

I propose a system akin to the market within the arena, but with a twist – the use of cryptocurrency to purchase items. These items could be delivered in a manner similar to the airdrops a few months ago or the hidden locations like the coke bricks in the ocean.

Rather than allowing direct selection of desired items, we could introduce black market packages that are airdropped:

Bronze (containing pistols and other illegal items)
Silver (including SMGs and 1-handed automatic weapons, along with similar illegal items)
Gold (containing 1-3 ARs, various random illegal items, chips, keys, etc.)
These are just quick examples for names. The airdrop system was previously implemented on the server a few months ago, sparking conflict between groups and allowing for investigation roleplay on the police side.

I acknowledge the similarity to Jim's concept, but this adds another dimension to the 'illegal currency' within the server."

### Reason

Adding another layer to the ability to get guns, chips, D-keys, etc. In the server other than Jim, roleplay consequences cut off possible dealing with weapons in roleplay, Jim being a very competitive server mechanic and him being bought out pretty often.

Cargo is also in place and supplies these however not everyone in the server is there to shoot. The same groups commonly reap the benefit of cargo while some of the newer groups get punished for losing items more than ever. Providing limited roleplay. 

another way to generate crypto currency is, Selling currency to other players. Maybe someone will try to control the black market. (Another possible gang process)

**Comment by ImArteria** [comment]
Can we revisit this? Feel like having something to spend Crypto on would give boosting something to do and give tradeoffs. I think this would help a lot.

**Comment by SluggsV2** [comment]
I back this fully! Would love to spend my cryp on things!

I like the black market airdrop idea

**Comment by Manmohan-zanwar** [comment]
Yeah black market would be a good idea and we aren't even expecting the stuff for normal price we can keep things on high stakes so people grind more for stuff.

**Comment by InZidiuZ** [comment]
This issue tracker is being shut down and replaced by Discord threads. If this issue is still relevant, create a thread on [Discord](https://discord.gg/opfw).
